### DISCUSSION TITLE
macos version 10.15 support?

### DISCUSSION THREAD

**Discussion by lucendio** [utmapp/UTM#2709] [UTM Help (macOS)] [answered]
I saw that UTM is actually build on [10.15](https://github.com/utmapp/UTM/blob/13664282a2a9fb239f62c5777cb45cabcce29fae/.github/workflows/build.yml#L25), on the other hand it appears that only [v11 is supported](https://github.com/utmapp/UTM/issues/2635). Can you share some insights as to what issues you ran into with 10.15? Maybe we can help to make it work. But then again it's probably not worth the effort since v12 is around the corner.

**Comment by osy** [comment, marked as answer]
It uses a lot of SwiftUI 2 which only works on macos 11.

**Comment by lucendio** [comment]
@osy  thank you for moving it. I should done this right away! I'm sorry.
